ngrinder-controller/src/main/java/org/ngrinder/infra/config/Config.java

```diff
@@ -735,4 +735,25 @@ public long getInactiveClientTimeOut() {
 	public boolean isEnableStatistics() {
 		return getControllerProperties().getPropertyBoolean(PROP_CONTROLLER_ENABLE_STATISTICS);
 	}
+
+	/**
+	 * Get Csv Separator.
+	 * defaultValue is comma.
+	 *
+	 * @return String csvSeparatorValue
+	 */
+	public String getCsvSeparator() {
+		String csvSeparatorValue = getControllerProperties().getProperty(
+				PROP_CONTROLLER_CSV_SEPARATOR, "comma");
+		if (csvSeparatorValue.equals("comma")) {
+			csvSeparatorValue = ",";
+		} else if (csvSeparatorValue.equals("tab")) {
+			csvSeparatorValue = "\t";
+		} else {
+			csvSeparatorValue = ";";

```

Comment:
default should be "," so that if wrong value is specified the csvSeparator should be ","
and semicolon should be explicitly specified.